sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/operator/BeamSqlUdfExpression.java

```diff
@@ -78,6 +79,7 @@ private void reConstructMethod() {
       for (String pc : paraClassName) {
         paraClass.add(Class.forName(pc));
       }
+      udfIns = Class.forName(className).newInstance();
```

Comment:
So for the static eval method version, this just returns the global static class instance? Or null? Or?